docs/fsharp-cheatsheet.md

```diff
@@ -414,7 +438,14 @@ Another way of implementing interfaces is to use *object expressions*.
 	    | Even -> printfn "%d is even" i
 	    | Odd -> printfn "%d is odd" i
 
-*Parameterized active patterns*:
+    let (|Phone|Email|) (s:string) =
+        if s.Contains '@' then Email $"Email: {s}" else Phone $"Phone: {s}"
```

Comment:
```suggestion
        if s.Contains '@' then Email s
        else Phone s
```

Reply:
I prefer the example as e.g. `Email $"Email: {s}"` to demonstrate that the argument to the identifier didn't need to match the parameter to the active pattern function.

Reply:
My main thing was that for two cases doing equal amounts of work to put the else on its own line
Ambivalent about putting the prefix redundantly in the DU body (normally I'd expect a print function to pattern match and layer that on)
But I think the example in general is worth having so all good